src/client/app/types/redux/groups.ts

```diff
@@ -18,6 +18,7 @@ export interface GroupData {
 	childMeters: number[];
 	childGroups: number[];
 	deepMeters: number[];
+	deepGroups: number[];
```

Comment:
This change means the group data is modified. Thus, where is it used may need changes. For example, I'm seeing a TypeScript  error in /usr/src/app/src/client/app/components/groups/EditGroupModalComponent.tsx on line 360 because there is an extra key (deepGroups) in the state. This can be fixed with an omit (currently done by submitState which is the older way OED did this). I note this may go away based on another comment about creating a new API endpoint.